entityDumper/types.go

```diff
@@ -12,6 +12,7 @@ type ChannelDumperOptions struct {
 	outputFolderAbsPath       string
 	MetadataOnly              bool
 	StartingDate              string
+	ConfigLabels []string
```

Comment:
Let's leave it for refactoring. For now I've renamed the struct to just `DumperOptions`. Otherwise there might be too many changes, not really connected to this PR, as we also want to have it written down into one .sql file here :smiley: